src/Doctors/App/Controllers/DeleteDoctorController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Doctors\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Doctors\Domain\Models\Doctor;
+
+final readonly class DeleteDoctorController
+{
+    public function __invoke(Doctor $doctor): JsonResponse
+    {
+        $doctor->delete();
+
+        return response()->json();
+    }
```

Comment:
This is more accurate since you're not returning any data

```suggestion
    public function __invoke(Doctor $doctor): Response
    {
        $doctor->delete();

        return response()->noContent();
    }
```